DOCS-3224: Add API reference notice page (#2088)
MultiSafepay/docs: content/api/api-ref/_index.md

---
weight: 10
meta_title: "API reference - Introduction - MultiSafepay Docs"

---

{{< code-block hide >}}
{{< /code-block >}}

{{< description >}}
<p align="center">
  <img src="/svgs/FAQ.svg" width="200px" position="center">
</p>
<br>

# Oops! Looking for our API reference? 
<hr class="separator">

> We've moved. And what's more, we're new and improved! [Check us out](https://docs-api.multisafepay.com/reference/introduction).

Reviewer: ```suggestion
It's moved. And what's more, it's new and improved! 
[Check it out](https://docs-api.multisafepay.com/reference/introduction)!
```
